app/views/saved_scenarios/show.html.haml

```diff
@@ -28,7 +28,41 @@
     %li
       %strong #{t('scenario.end_year')}:
       = @scenario.end_year
+  .sharing_links
+    = t('scenario.share.by')
+    .copy_link
+      %a
+        .icon{ class: 'fa fa-copy' }
+        Copy link
+      .copy_box
+        .arrow
+        .cross{ class: 'fa fa-times', title: 'Close' }
+        .url= text_field_tag 'url', request.original_url
+
+    = link_to share_by_mail_link do
+      .icon{ class: 'fa fa-envelope-o' }
+      Email
+    = link_to share_by_twitter_link, open_attributes do
+      .icon{ class: 'fa fa-twitter' }
+      Twitter
+    = link_to share_by_linkedin_link, open_attributes do
+      .icon{ class: 'fa fa-linkedin' }
+      LinkedIn
+    = link_to share_by_facebook_link, open_attributes do
+      .icon{ class: 'fa fa-facebook-official' }
+      Facebook
 
   %p
     %small{:style => "font-size: 12px; font-weight: normal"}
       = t("scenario.overwritten")
+
+:javascript
```

Comment:
Not a huge fan of javascript in HTML templates. might be best to move this to its own file.